tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -12239,4 +12239,231 @@ TEST_F(HVACVRFFixture, VRF_FluidTCtrl_ReportOutputVerificationTest)
+        "    1.0,                     !- Coefficient2 x",
+        "    0.0,                     !- Coefficient3 x**2",
+        "    0.0,                     !- Coefficient4 x**3",
+        "    0,                       !- Minimum Value of x",
+        "    1,                       !- Maximum Value of x",
+        "    ,                        !- Minimum Curve Output",
+        "    ,                        !- Maximum Curve Output",
+        "    Dimensionless,           !- Input Unit Type for X",
+        "    Dimensionless;           !- Output Unit Type",
+
+        "  Curve:Cubic,",
+        "    HRCAPFTHeat,             !- Name",
+        "    0.8,                     !- Coefficient1 Constant",
+        "    0.0,                     !- Coefficient2 x",
+        "    0.0,                     !- Coefficient3 x**2",
+        "    0.0,                     !- Coefficient4 x**3",
+        "    0,                       !- Minimum Value of x",
+        "    1,                       !- Maximum Value of x",
+        "    ,                        !- Minimum Curve Output",
+        "    ,                        !- Maximum Curve Output",
+        "    Dimensionless,           !- Input Unit Type for X",
+        "    Dimensionless;           !- Output Unit Type",
+
+        "  Curve:Cubic,",
+        "    HREIRFTHeat,             !- Name",
+        "    0.9,                     !- Coefficient1 Constant",
+        "    0.0,                     !- Coefficient2 x",
+        "    0.0,                     !- Coefficient3 x**2",
+        "    0.0,                     !- Coefficient4 x**3",
+        "    0,                       !- Minimum Value of x",
+        "    1,                       !- Maximum Value of x",
+        "    ,                        !- Minimum Curve Output",
+        "    ,                        !- Maximum Curve Output",
+        "    Dimensionless,           !- Input Unit Type for X",
+        "    Dimensionless;           !- Output Unit Type",
+    });
+
+
+
+    ASSERT_TRUE(process_idf(idf_objects));
+    CurveManager::GetCurveInput();
+
+    int VRFCond = 1;
+    VRF.allocate(1);
+    VRF(VRFCond).CondenserNodeNum = 0;
+    VRF(VRFCond).CondenserType = 1; // DataHVACGlobals::AirCooled
+    VRF(VRFCond).ZoneTUListPtr = 1;
+    VRF(VRFCond).CoolingCapacity = 20000.0;
+    VRF(VRFCond).HeatingCapacity = 20000.0;
+    VRF(VRFCond).CoolingCOP = 3.0;
+    VRF(VRFCond).HeatingCOP = 3.0;
+    VRF(VRFCond).RatedCoolingPower = VRF(VRFCond).CoolingCapacity / VRF(VRFCond).CoolingCOP;
+    VRF(VRFCond).RatedHeatingPower = VRF(VRFCond).HeatingCapacity / VRF(VRFCond).HeatingCOP;
+    VRF(VRFCond).PipingCorrectionCooling = 1.0;
+    VRF(VRFCond).PipingCorrectionHeating = 1.0;
+    // Curve Indices, including HR curves (3 & 4)
+    VRF(VRFCond).CoolCapFT = 1;
+    VRF(VRFCond).CoolEIRFT = 1;
+    VRF(VRFCond).CoolEIRFPLR1 = 2;
+    VRF(VRFCond).HeatEIRFPLR1 = 2;
+    VRF(VRFCond).HRCAPFTHeat = 3;
+    VRF(VRFCond).HREIRFTHeat = 4;
```

Comment:
Assigned HR Curves (important). HRCAPFTHeat is a constant 0.8, and HREIRFTHeat is a constant 0.9